test/PerfTests/JavaScript/ManyInstancesOrchestrator/index.js

```diff
@@ -0,0 +1,21 @@
+﻿/*
+ * This function is not intended to be invoked directly. Instead it will be
+ * triggered by an HTTP starter function.
+ * 
+ * Before running this sample, please:
+ * - create a Durable activity function (default name is "Hello")
+ * - create a Durable HTTP starter function
+ * - run 'npm install durable-functions' from the wwwroot folder of your 
+ *    function app in Kudu
+ */
+
+const df = require("durable-functions");
+
+module.exports = df.orchestrator(function* (context) {
+    const outputs = [];
+
+    for(var i = 0; i < 100; i++){
+        yield context.df.callActivity("Hello", "Tokyo");
```

Comment:
Instead of hardcoding the input to "Tokyo", use the value of `i` as part of the input. It will make debugging easier.